src/compiler/tsbuildPublic.ts

```diff
@@ -328,6 +326,7 @@ function getCompilerOptionsOfBuildOptions(buildOptions: BuildOptions): CompilerO
     commonOptionsWithBuild.forEach(option => {
         if (hasProperty(buildOptions, option.name)) result[option.name] = buildOptions[option.name];
     });
+    result.tscBuild = true;

```

Comment:
How will this flag affect people using our APIs to implement build mode like tsc? Will it be set for them? Or can we just assume it's always true?

Reply:
Alrighty; just verifying so people are not surprised.